buildspec.yml

```diff
@@ -1,11 +1,26 @@
-version: 0.1
+# CI always looks for this string: PRX_
+version: 0.2
+env:
+  variables:
+    DOVETAIL_HOST: "dovetail.staging.prxu.org"
+    DOVETAIL_PROD_HOST: "dovetail.prxu.org"
+    FEEDER_HOST: "feeder.staging.prx.tech"
+    METRICS_HOST: "metrics.staging.prx.tech"
+    PUBLISH_HOST: "publish.staging.prx.tech"
+    PUBLISH_USER: "test@test.com"
+    PUBLISH_PASS: "test1234"
```

Comment:
Our precious password!  Checked into source control!

(Maybe should use parameterstore on this one, just for good measure).